src/ragger/navigator/nano.py

```diff
@@ -0,0 +1,52 @@
+"""
+   Copyright 2022 Ledger SAS
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+from typing import List
+
+from ragger.backend import BackendInterface
+from ragger.firmware import Firmware
+from ragger.navigator import NavigatorInterface, NavigationInstruction
+
+
+class NanoNavigator(NavigatorInterface):
+
+    _ARGS_KEY = 'args'
+
+    def __init__(self, backend: BackendInterface, firmware: Firmware):
```

Comment:
I think this `__init__` is superfluous, as it just redirect the argument to it motherclass, in the exact same order.